### DISCUSSION TITLE
is it possible to migrate from open ethereum v3.x to erigon?

### DISCUSSION THREAD

**Discussion by donamk** [erigontech/erigon#4579] [Q&A]
I am running three open ethereum nodes using POA.

I have searched a lot but I couldn't find an article about how to migrate and it is possible.

Is it possible to migrate from open ethereum v3.x to erigon? If the answer is yes, how can I do that?

**Comment by dvdblk** [comment]
Same question here. We have a snapshot of almost fully synced node and would like to use this as a starting point for an erigon node.

**Comment by AskAlexSharov** [comment]
we do support `trace_*` rpc methods for compatibility with open ethereum
